tests/workflows/theiacov/test_wf_theiacov_illumina_pe.yml

```diff
@@ -134,10 +134,6 @@
     - path: miniwdl_run/call-read_QC_trim/call-ncbi_scrub_pe/stdout.txt
     - path: miniwdl_run/call-read_QC_trim/call-ncbi_scrub_pe/task.log
       contains: ["wdl", "illumina_pe", "ncbi_scrub", "done"]
-    - path: miniwdl_run/call-read_QC_trim/call-ncbi_scrub_pe/work/FWD_SPOTS_REMOVED
-      md5sum: 897316929176464ebc9ad085f31e7284
-    - path: miniwdl_run/call-read_QC_trim/call-ncbi_scrub_pe/work/REV_SPOTS_REMOVED
-      md5sum: 897316929176464ebc9ad085f31e7284
```

Comment:
FYI some of these CI updates in theiacov_illumina_PE wf and theiacov_clearlabs wf were required as they were missed over on PR #527 . ILMN PE and ClearLabs tests were not triggered in the CI and missed on accident.

@cimendes @jrotieno hopefully these CI changes are not unexpected? Can you check to make sure it's plausible & expected for these hashes & string searches to be changed?

Reply:
This is an expected change as now reads are interleaved prior to scrubbing, so the number of spots removed includes information for all reads. This was done to ensure that if a read is removed, so is its mate. @kapsakcj if you want to still verify this behaviour in the CI, you can change it to `miniwdl_run/call-read_QC_trim/call-ncbi_scrub_pe/work/SPOTS_REMOVED` and verify the md5sum of that.